### DISCUSSION TITLE
Bug in mobile application page=cannot use camera

### DISCUSSION THREAD

**Discussion by GuyverMack** [community/community#109060] [GitHub Education]
### Hi everyone,

The upload proof page does not allow uploading of scanned images, only webcam pics, but I do not have a webcam with high enough resolution. So I opened the page on iPhone in Safari so at least I could use my camera to photograph the transcript.
It defaults to use the front camera (looking at me) instead of the rear one! I switched cameras using the dropdown menu, but there is no preview image shown for the rear camera, only the front one. That means I have to take pictures of my transcript blindly without being able to see what I'm capturing. The only way I can send a clear, HQ document is to scan it, but uploading isn't possible.
This is a bug, in that the rear camera doesn't show a preview image. In order to apply, I need the page to accept uploaded images, or to show a preview for the camera.
I tried uploading a new scan of the same doc, but the page won't let me resubmit it.

**Comment by aradiyo** [comment]
Same problem 

Please help me as well
